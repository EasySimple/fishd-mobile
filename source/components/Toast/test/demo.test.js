import webDemoTest from '../../../tests/demoTest';

webDemoTest('Toast');
