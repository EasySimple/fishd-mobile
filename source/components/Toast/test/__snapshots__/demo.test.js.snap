// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Toast demo correctly 1`] = `
<div>
  <div
    class="components-toast-demo"
  >
    <a
      aria-disabled="false"
      class="fm-button fm-button-default fm-button-normal"
      role="button"
    >
      <span>
        文 本
      </span>
    </a>
    <br />
    <a
      aria-disabled="false"
      class="fm-button fm-button-default fm-button-normal"
      role="button"
    >
      <span>
        loading
      </span>
    </a>
    <br />
    <a
      aria-disabled="false"
      class="fm-button fm-button-default fm-button-normal"
      role="button"
    >
      <span>
        success
      </span>
    </a>
    <br />
    <a
      aria-disabled="false"
      class="fm-button fm-button-default fm-button-normal"
      role="button"
    >
      <span>
        error
      </span>
    </a>
    <br />
    <a
      aria-disabled="false"
      class="fm-button fm-button-default fm-button-normal"
      role="button"
    >
      <span>
        network failure
      </span>
    </a>
    <br />
    <a
      aria-disabled="false"
      class="fm-button fm-button-default fm-button-normal"
      role="button"
    >
      <span>
        自定义
      </span>
    </a>
  </div>
</div>
`;
